robottelo/host_helpers/satellite_mixins.py

```diff
@@ -468,6 +468,9 @@ def configure_insights_on_prem(self, username=None, password=None, registry=None
         self.register_to_cdn()
         self.setup_rhel_repos()
         self.setup_satellite_repos()
+        # Set IPv6 system proxy on Satellite, to reach container registry
+        self.enable_ipv6_system_proxy()
```

Comment:
The installer deletes those in https://github.com/theforeman/foreman-installer/blob/55c6d3d5f02fcd05fcdb0f3cb75cf4e245c9e9e3/hooks/boot/05-environment.rb#L2, so podman still uses non-proxy path :/

Reply:
I think the question here is if the setting is generic for all containers in user's environments, or just for "our" containers. Personally I would think that we need to set it specifically for our containers, without telling podman how to access the outside environment in general.

Reply:
Now that I think again about it, you are right. We can set it for the whole environment then. :+1: from me to option 2.

Reply:
I tried the provided solution and setting `env` under `[containers]` doesn't work as its part of ENGINE TABLE check `man containers.conf`, so instead we need to set this separately in containers.conf, which works correctly if we prefer 2nd solution
```
[containers]
http_proxy = true

[engine]
env = ["HTTPS_PROXY=<proxy-url>"]
```

Reply:
we agreed offline to keep only env in [engine], and we don't want containers to connect to internet, so no [containers] is required